### ISSUE TITLE
Merge with NRCHKB?

### ISSUE THREAD

**Issue by crxporter** [RaymondMouthaan/node-red-homekit-docker#38]
Calling @shaquu to the discussion 

We've been talking about putting this docker version into the NRCHKB organization and wondered what you (Ray) thought about this idea? We can manage them together as a group. I'm not sure if there are any downsides but it could be nice to everything together.

**Comment by Shaquu** [comment]
@RaymondMouthaan as you said you are not using docker for NRCHKB anymore.
We value you work a lot. You are reacting to issues and new versions fast and we appreciate it.

Our idea is:
 - move you repository under NRCHKB Organization
 - prioritise docker way of using NRCHKB as we believe this should be a recommended way

**Comment by crxporter** [comment]
PS: we don't want to kick you out! We'd like you to be part of the NRCHKB organization and keep contributing as much as you'd like

**Comment by RaymondMouthaan** [comment]
Hi @crxporter & @Shaquu, 

Merging the docker project in to NRCHKB was initially my whole idea. First to make it work from my repo and then merge it into the  NRCHKB project. Even have a nice docker repo at docker hub instead of one of mine. 

This same approach was done for the official Node-RED docker images as well and nowadays that works fine.

If you like me to maintain the docker part of the project, I am willing to do so and need access to the GitHub project. Also we need an account on travis, since currently all images are build by travis automatically and pushed to Docker hub. 
 
We could also consider to add dev / test version of NRCHKB and publish them to Docker hub before releasing, that could be handy. 

So basically, yes I am up to the idea of merging both projects into one. One disadvantage is that all current users of my docker images need to change `raymondmm/node-red-homekit` into `<new_repo>/node-red-homekit`. Once both projects are merged and images are available on the new (official) location, I'll put a note on my GitHub project and Docker Hub that these are deprecate with a link to the new location. Eventually I'll remove both.

**Comment by RaymondMouthaan** [comment]
The merge is done successfully.
